brambling/tests/functional/test_organizer_views.py

```diff
@@ -238,6 +239,42 @@ def test_unicode_csv(self):
         self.assertIn('£200.00', content[1])
 
 
+class AttendeeFilterViewTest(TestCase):
+    def setUp(self):
+        self.event = EventFactory(collect_housing_data=True, currency='GBP')
```

Comment:
Do these attributes (collect_housing_data and currency) need to be set for this test to work?